lib/OpenQA/CacheService/Client.pm

```diff
@@ -71,6 +73,10 @@ sub enqueue ($self, $request) {
     return undef;
 }
 
+sub _is_ipv4_addr ($ip) {
+    return $ip =~ /^(\d{1,3})\.(\d{1,3})\.(\d{1,3})\.(\d{1,3})$/;
+}
```

Comment:
We already use `Regexp::Common`, so you can use https://metacpan.org/pod/Regexp::Common::net.